### PR TITLE
ClangFormat: Update config

### PR DESCRIPTION
**Pull request by uklotzde**

### CODE REVIEW

File: .clang-format [hunk 1]
```diff
@@ -8,17 +9,17 @@ UseTab:       Never
...
 # Dumping the options of the base style for comparison:
 # clang-format -style=google -dump-config > .clang-format_google
-Language:     Cpp
 AccessModifierOffset: -2
 AlignAfterOpenBracket: DontAlign
 AlignOperands: false
 AllowShortFunctionsOnASingleLine: None
 AllowShortIfStatementsOnASingleLine: false
 AllowShortLoopsOnASingleLine: false
-AlwaysBreakBeforeMultilineStrings: true
```
Comment (Holzhaus) [inline .clang-format:21]: Why did you change this?

Comment (Swiftb0y) [reply .clang-format:21]: We only overwrite options that differ from the base-style. `AlwaysBreakBeforeMultilineStrings: true` is already part of the google style we inherit from.

Comment (uklotzde) [reply .clang-format:21]: The commit message refers to the advice given in the comment a few lines above.

**Review by Swiftb0y — APPROVED** [review]
